<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629281300410' stop='1629281600586'>
    <name>delete function</name>
    <title>delete function</title>
    <test-cases>
        <test-case start='1629281300563' status='failed' stop='1629281600585'>
            <name>Delete the added projects.</name>
            <title>Delete the added projects.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='4db7acb5-cdf2-4e4a-81e0-f0919d4d8f1b-attachment.png' type='image/png' size='74551'/>
                <attachment title='Screenshot' source='98efc44b-1e83-4084-9ca0-5cd3191fc612-attachment.png' type='image/png' size='60889'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (E:\vmware\Loop\GitHub\e2e\node_modules\jasmine\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:4281:23)
                    at listOnTimeout (internal/timers.js:554:17)
                    at processTimers (internal/timers.js:497:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>